samples/contrib/e2e-outlier-drift-explainer/seldon/README.md

```diff
@@ -0,0 +1,75 @@
+# Seldon End to End Deployment Examples
+
+ * [CIFAR10 Image Classifier with Explainer, Outlier detector and drift detector](#cifar10-image-classification-model)
+ * [Income Classifier with Explainer and Outlier Detector](#income-classification-model)
+
+## Running Pipelines
+
+ * [See "tested on" section](../README.md#tested-on)
+
+## Running in Kubeflow Jupyter Lab
+
+ 1. Create a new JupyterLab server using the image `seldonio/jupyter-lab-alibi-kale:0.11`
+ 
+
+## CIFAR10 Image Classification Model
+
+ * Seldon CIFAR10 Image Model with deployed explainer, outlier detector and drift detector
+    * [Kale annotated Jupyter Notebook](./seldon_e2e_cifar10.ipynb)
+    * [Pipeline](./seldon_e2e_cifar10.kale.py)
+      * Assumes `storage_class="nfs-client"`
+
+![pipeline](cifar10-pipeline.png)
+
+
+### Model Deployment and Predictions
+
+Deploy a trained tensorflow model and call the Seldon API to get predictions.
```

Comment:
```suggestion
Deploy a trained Tensorflow model and call the Seldon API to get predictions.
```

Reply:
Can this TF model be used with KFServing as well?

Reply:
Yes, this was a mistake I forgot to add files. Thanks for spotting!